pkg/cmd/pr/checks/checks.go

```diff
@@ -37,7 +37,7 @@ func NewCmdChecks(f *cmdutil.Factory, runF func(*ChecksOptions) error) *cobra.Co
 	}
 
 	cmd := &cobra.Command{
-		Use:   "checks",
+		Use:   "checks {<number> | <url> | <branch>}",
```

Comment:
Hi, since the argument is optional (as opposed to required), can you change `{}` braces to `[]` brackets? Thank you! https://github.com/cli/cli/blob/trunk/docs/command-line-syntax.md#optional-arguments

Reply:
Thanks @mislav for pointing this out, I have changed the argument type.